docker/builder/Dockerfile

```diff
@@ -118,6 +118,16 @@ RUN wget -nv https://github.com/fastobo/fastobo-validator/archive/refs/tags/v0.4
     cd /build && \
     rm -rf fastobo-validator-0.4.0 fastobo-validator-0.4.0.tar.gz /root/.cargo
 
+# Compile rdftab.
+RUN wget -nv https://github.com/ontodev/rdftab.rs/archive/refs/tags/v0.1.1.tar.gz \
```

Comment:
Can we extract the version here as we usually do? Just to make updating to later versions easier.

Reply:
$RDFTAB_VERSION

Reply:
From my experience it’s not that useful when building from source, as updating to a later version most often requires some work anyway beyond merely changing the version number. But sure.

Reply:
Usually I agree, but because of the git diff not showing lines around, it is very hard to review a pull request "bumped version number of X" - you always then have to "view file" if you want to be sure they bumped it in all places..